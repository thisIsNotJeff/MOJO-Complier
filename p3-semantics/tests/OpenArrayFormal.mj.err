 line 1, column 9: warning: open array passed by value (a)
