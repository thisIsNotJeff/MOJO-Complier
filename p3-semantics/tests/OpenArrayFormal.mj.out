BEGIN _BUILTIN
_BUILTIN.int=int
_BUILTIN.bool=bool
_BUILTIN.false:bool=false
_BUILTIN.true:bool=true
_BUILTIN.Null=^null
_BUILTIN.nil:Null=0
_BUILTIN.Refany=^null
_BUILTIN.Address=^null
_BUILTIN.Root=null
object {
}
_BUILTIN.Text=^null
_BUILTIN.First:(...)
_BUILTIN.Last:(...)
_BUILTIN.New:(...)
_BUILTIN.Number:(...)
BEGIN OpenArrayFormal
OpenArrayFormal.foo:(VALUE a:[null]int)
BEGIN OpenArrayFormal.foo
OpenArrayFormal.foo.a:[null]int
END OpenArrayFormal.foo
END OpenArrayFormal
END _BUILTIN
